builtin/providers/aws/config.go

```diff
@@ -121,8 +118,11 @@ func (c *Config) Client() (interface{}, error) {
 		log.Println("[INFO] Initializing RDS Connection")
 		client.rdsconn = rds.New(awsConfig)
 
+		awsKinesisConfig := awsConfig
```

Comment:
Since ``awsConfig`` is a pointer I think you're just making a new reference to the same struct here and then mutating it, rather than copying it as I think you intended.